src/libraries/System.ComponentModel.TypeConverter/tests/ILLink.Descriptor.xml

```diff
@@ -0,0 +1,21 @@
+<linker>
+  <assembly fullname="TestUtilities">
+   	<namespace fullname="System" />
+        <type fullname="System.PlatformDetection" />
+  </assembly>
+  <assembly fullname="System" />
+  <assembly fullname="xunit.runner.utility.netcoreapp10" />
+  <assembly fullname="Microsoft.DotNet.XUnitExtensions"/>
```

Comment:
Out of curiosity, are there specific parts of these assemblies that show up in `System.ComponentModel.TypeConverter` and need to not be linked out? Are `xunit` specific assemblies not only related to the test runner? (As in why is having these in `eng/testing/ILLink.Descriptor.xunit.xml` not sufficient?)

Reply:
I think it is actually redundant. I will move them into ILLink.Descriptor.xunit.xml